_data/learning.yml

```diff
@@ -26,6 +26,7 @@ books:
       - link: /learn/quickstart/arrays_strings
       - link: /learn/quickstart/operators_control_flow
       - link: /learn/quickstart/organising_code
+      - link: /learn/quickstart/derived_types
```

Comment:
I would put the link  before "operators_control_flow" to keep a logical order

Reply:
I put it last on purpose since it makes use of modules, subroutines, and functions that are covered in the organising_code page.

Reply:
Actually yes that's a good point, I agree with @smeskos

Reply:
Good point indeed.